test/tester2000.js

```diff
@@ -12,6 +12,14 @@ const client = new Client({
   makeCache: Options.cacheWithLimits({
     MessageManager: 10,
     PresenceManager: 10,
+    UserManager: {
+      maxSize: 1,
+      keepOverLimit: v => v.id === client.user.id,
```

Comment:
We should probably make this the default if possible.. @discordjs/the-big-3 thoughts?

Reply:
No, we don't even have a default on capping users/members.

Reply:
I think all structures in a cached manager have a reference to the client, making this additional parameter (and, to some extend, complexity redundant).
I am not sure I really follow your comment however. 😅

Reply:
Ok ignore me, i'm dumb